compose/foundation/foundation/src/skikoTest/kotlin/androidx/compose/foundation/gestures/SkikoLazyListTest.kt

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.foundation.gestures
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.LazyListState
+import androidx.compose.foundation.text.BasicTextField
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.input.pointer.pointerInput
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.test.ExperimentalTestApi
+import androidx.compose.ui.test.ScrollWheel
+import androidx.compose.ui.test.onNodeWithTag
+import androidx.compose.ui.test.performMouseInput
+import androidx.compose.ui.test.runComposeUiTest
+import androidx.compose.ui.unit.dp
+import kotlin.test.Test
+import kotlin.test.assertTrue
+
+@OptIn(ExperimentalTestApi::class)
+class SkikoLazyListTest {
+    // https://github.com/JetBrains/compose-multiplatform/issues/3559
+    @Test
+    fun textFields() = runComposeUiTest {
```

Comment:
This test passes even without a fix commit https://github.com/JetBrains/compose-multiplatform-core/commit/d02e1c5c9b79cd481bb57b23ec751b03ce612fcb

Although enabled switches betwen true/false.

Is it expected?

Reply:
I'm not sure what's going on. I tried to change the test similar to the snippet in the bug report:

```kotlin
    @Test
    fun textFields() = runComposeUiTest {
        val state by mutableStateOf(LazyGridState())
        setContent {
            LazyVerticalGrid(
                state = state,
                columns = GridCells.Fixed(4),
                modifier = Modifier.testTag("list").fillMaxSize()
            ) {
                items((0 until 200).toList()) { item ->
                    BasicTextField(
                        value = "Item: $item",
                        enabled = item % 2 == 0,
                        onValueChange = {},
                    )
                }
            }
        }

        onNodeWithTag("list").performMouseInput {
            scroll(1000f, ScrollWheel.Vertical)
        }
        runOnIdle { assertTrue(state.firstVisibleItemIndex > 0) }
    }
    ```
    
    But it passes without the fix commit anyway.

Reply:
Fixed. maybe performMouseInput scrolls to-non crashable state